src/me.rs

```diff
@@ -216,6 +217,38 @@ mod native {
   }
 }
 
+#[derive(Debug, Clone)]
+pub struct FrameMotionVectors {
+  mvs: Box<[MotionVector]>,
```

Comment:
I think this is equivalent structurally to a `Vec`, but has less flexibility. Is there a specific reason we would need to store it this way?

Reply:
The array size is constant (cols × rows), this avoids to expose methods which could resize the array (like `.push()`).

Reply:
It also has the additional bonus of being slightly smaller and if we have plenty of motion vectors it could have some impact :)